src/content/docs/style-guide/writing-strategies/Untitled-1

```diff
@@ -0,0 +1,29 @@
+---
+title: OpenTelemetry examples
+metaDescription: "Our stance on OpenTelemetry examples moving forward."
+---
+
+This page outlines the docs team's approach to documenting OpenTelemetry examples.
+
+## Background [#background]
+
+In New Relic documentation for OpenTelemetry (OTEL), we provide examples aimed to demonstrate the most common configurations we expect users to use. Historically, the docs site contained these examples on the docs site. However, in May 2024, a joint effort with Relics working on OTEL, we decided to migrate these examples from the docs site to the [newrelic-opentelemetry-examples](https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main) repo. For example, here’s a PR where we moved all the infrastructure examples: [PR #17742](https://github.com/newrelic/docs-website/pull/17742).
+
+## Why we support this change [#why-we-support-this-change]
+
+The reasons for this migration are:
+* Reduce maintenance burden: Both the OTEL and docs team are small, with a lot of surface area. We have to keep our processes lean to stay afloat.
+* Reduce redundancy across both the repo and docs site: Keeping multiple copies encourages (and almost guarantees) diverging sources of truth and bit rot.
+* Make it easier to consume the examples: Writing examples with text, like “Put X content in a file call Y”, is a bad developer experience since it's time consuming and error prone. It's much better to clone a repo, update an environment variable or two, and run a command.
```

Comment:
I'd move the these last two bullets to the top and move "reduce maintenance burden" to the bottom of the list